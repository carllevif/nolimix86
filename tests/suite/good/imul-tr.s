  .text
imul %t0, %eax
