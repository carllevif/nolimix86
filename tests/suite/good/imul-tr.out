  .text
start:
imul %t0, %eax
